src/Suave/Owin.fs

```diff
@@ -634,8 +634,17 @@ module OwinApp =
         HttpContext.request_ >--> HttpRequest.headers_,
         HttpContext.response_ >--> HttpResult.headers_
 
+      let handleResponse  (rhs : Dictionary<string, string[]>) = 
+        let handleKey key =
```

Comment:
Should there be a const in Cookies.fs for "Set-Cookie" since it's being used around a lot?